lib/widgets/payment_request_dialog.dart

```diff
@@ -127,7 +96,45 @@ class PaymentRequestDialogState extends State<PaymentRequestDialog>
         .listen((settings) => _accountSettings = settings);
 
     _sentPaymentResultSubscription = widget.accountBloc.fulfilledPayments.listen((fulfilledPayment) {
-      // Trigger the collapse animation and show flushbar 200ms after the animation is completed
+      controller = AnimationController(
+          vsync: this, duration: Duration(milliseconds: 600));
+      borderAnimation = Tween<double>(begin: 0.0, end: 8.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      opacityAnimation = Tween<double>(begin: 0.0, end: 1.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      colorAnimation = new ColorTween(
+        begin: theme.BreezColors.blue[500],
+        end: theme.BreezColors.white[500],
+      ).animate(controller)
+        ..addListener(() {
+          setState(() {});
+        });
+
+      RenderBox _paymentTableBox = widget.tableKey.currentContext
```

Comment:
How the animation will work on the first item when the key is not used and not related to any widget?

Reply:
It won't work at the moment. As I mentioned earlier, findRenderObject() does not return a RenderBox when used on table so we can calculate it's size/position.

Reply:
We need a solution for this case as well, it is the first payment the user does...
One solution is to add a "dummy" sizebox with size 0 only to get its location instead of the table.